### ISSUE TITLE
Issues upgrading to v5.11.1

### ISSUE THREAD

**Issue by Xartos** [UpCloudLtd/terraform-provider-upcloud#647]
# The problem

When I try to use the module version `5.11.1` from version `5.10.1` I get the following error

```
│ Error: Unable to Upgrade Resource State
│ 
│   with module.kubernetes.upcloud_loadbalancer_backend.lb_backend["http"],
│   on modules/kubernetes-cluster/main.tf line 604, in resource "upcloud_loadbalancer_backend" "lb_backend":
│  604: resource "upcloud_loadbalancer_backend" "lb_backend" {
│ 
│ This resource was implemented without an UpgradeState() method, however Terraform was expecting an implementation for version 0 upgrade.
│ 
│ This is always an issue with the Terraform Provider and should be reported to the provider developer.
```

According to the error this is some issue on your side

**Comment by villevsv-upcloud** [comment]
Hello and thanks for reporting the bug.

We have identified the cause for the problem and will issue a fix.

We will keep this issue updated accordingly to let you know once there is a new release out.

**Comment by villevsv-upcloud** [comment]
Hi, this is now fixed in the release [v5.11.2](https://github.com/UpCloudLtd/terraform-provider-upcloud/releases/tag/v5.11.2). I will be closing the issue as completed.
